Description: evas: fix double free.
 This fixes a double free in evas that was causing test failures
Author: Cedric BAIL <cedric@osg.samsung.com>
Applied-Upstream: 11ac6b145d826159b6d336964a965d330127c4cb
Last-Update: 2016-01-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/tests/evas/evas_test_textblock.c
+++ b/src/tests/evas/evas_test_textblock.c
@@ -2195,10 +2195,6 @@
    fail_if((tr->x != tr2->x) || (tr->y != tr2->y) || (tr->w != tr2->w) ||
            (tr->h != tr2->h));
 
-   EINA_LIST_FREE(rects, tr)
-      free(tr);
-   EINA_LIST_FREE(rects2, tr2)
-      free(tr2);
    eina_iterator_free(it);
    eina_iterator_free(it2);
 
@@ -2308,10 +2304,6 @@
    tr3 = eina_list_nth(rects, 3);
    fail_if((tr2->x != tr3->x));
 
-   EINA_LIST_FREE(rects, tr)
-      free(tr);
-   EINA_LIST_FREE(rects2, tr2)
-      free(tr2);
    eina_iterator_free(it);
    eina_iterator_free(it2);
 
@@ -2327,8 +2319,6 @@
 
    fail_if(eina_list_count(rects) != 2);
 
-   EINA_LIST_FREE(rects, tr)
-      free(tr);
    eina_iterator_free(it);
 
    /* Same run different styles */
@@ -2342,8 +2332,6 @@
 
    fail_if(eina_list_count(rects) != 3);
 
-   EINA_LIST_FREE(rects, tr)
-      free(tr);
    eina_iterator_free(it);
 
    /* Bidi text with a few back and forth from bidi. */
@@ -2358,8 +2346,6 @@
 
    ck_assert_int_eq(eina_list_count(rects), 3);
 
-   EINA_LIST_FREE(rects, tr)
-      free(tr);
    eina_iterator_free(it);
 
    END_TB_TEST();
